appengine/hello-world/standard/package.json

```diff
@@ -15,15 +15,17 @@
   "scripts": {
     "deploy": "gcloud app deploy",
     "start": "node app.js",
-    "system-test": "repo-tools test app",
+    "system-test": "mocha --exit test/*.test.js",
```

Comment:
fun idea which you can take or leave, if you add `c8` as a dependency, it turns on Node's V8 coverage collection, so the user gets:

![coverage](https://user-images.githubusercontent.com/194609/61314150-cc965e00-a7b0-11e9-8358-2179f0341413.png)

also understand if you think this is too much information for the user all at once.

Reply:
Trying to keep dependencies as light as possible.

Reply:
But lol, that coverage looks good!